openhtf/output/web_gui/__init__.py

```diff
@@ -408,7 +409,14 @@ def __init__(self, discovery_interval_s, disable_discovery, http_port,
         self.handle_test_state_update,
     )
 
-    path = BUILD_PATH if os.path.exists(BUILD_PATH) else PREBUILT_PATH
+    if 'custom_frontend' in conf:
+      path = conf.custom_frontend
+    else:
+      path = BUILD_PATH if os.path.exists(BUILD_PATH) else PREBUILT_PATH
+    if not os.path.exists(path):
+      raise ValueError('Frontend path `%s` does not exist.' % path)
+    _LOG.info('Running frontend from path `%s`')
+
```

Comment:
it feels like this logic should actually live in __main__ and the path to use should be passed into this server. Then it could even be an argument instead of a config variable, which makes more sense to me for some reason